inst/@sym/isconstant.m

```diff
@@ -24,12 +25,20 @@
 %% Example:
 %% @example
 %% @group
+%% @c doctest: +SKIP_UNLESS(pycall_sympy__ ('return Version(spver) < Version("1.10")'))
 %% syms x y
 %% A = [x 1 pi; 2 2*y catalan()]
 %%   @result{} A = (sym 2×3 matrix)
 %%       ⎡x   1      π   ⎤
 %%       ⎢               ⎥
 %%       ⎣2  2⋅y  Catalan⎦
+%% @c doctest: +SKIP_UNLESS(pycall_sympy__ ('return Version(spver) >= Version("1.10")'))
+%% syms x y
+%% A = [x 1 pi; 2 2*y catalan()]
+%%   @result{} A = (sym 2×3 matrix)
+%%      ⎡x   1   π⎤
+%%      ⎢         ⎥
+%%      ⎣2  2⋅y  G⎦
```

Comment:
yes right idea, but this `@c doctest:` is not a conditional include.

The primary point of the docs is to be human-readable (`help @sym/constant`).

I think you should just have the `>=` case.  That is, in most cases, we assume the reader is using the latest SymPy.

(This is somewhat dealt with on [the wiki](https://github.com/cbm755/octsympy/wiki/Development-Guide), but probably not clearly enough.)